NEWS

```diff
@@ -1,3 +1,7 @@
+New in version 0.3.3:
+- Rework broken LaTeX vignettes into RMarkdown, removing LaTeX dependency.
+- Add R CMD check to GitHub actions 
```

Comment:
I'm not sure if we need to mention CI and GH Actions here as they don't constitute new funtionality in the packge